projects/1.19/assets/no-chat-reports/nochatreports/lang/zh_cn.json

```
@@ -0,0 +1,130 @@
+{
+	"modmenu.descriptionTranslation.nochatreports": "将玩家消息中的用户签名去除，以使得其余玩家无法将此信息与玩家的Mojang/Microsoft账号关联，进而无法对其进行聊天举报。",
+	"gui.nochatreports.secure_chat": "由于安装有No Chat Reports模组，该选项无法实现，因为聊天消息中的签名已被移除。",
+	"gui.nochatreports.no_reporting": "由于在服务端安装有No Chat Reports，该服务器上无法进行聊天举报。",
+	"disconnect.nochatreports.client": "模组被配置为要求所连接的服务器安装No Chat Reports模组，但该服务器并未安装。",
+	"gui.nochatreports.safety_status_button.controls": "左击该按钮以切换签名策略",
+	"gui.nochatreports.safety_status.secure": "该服务器安装了No Chat Reports模组，或者以其余可识别的方式禁用了聊天举报。在此可以安全地进行聊天。",
```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |     |

**不要“进行”，能用动词就用**

known 翻译成“可识别的”有什么讲究吗

```suggestion
	"gui.nochatreports.safety_status.secure": "该服务器安装了No Chat Reports模组，或者用其他可识别的方式禁用了聊天举报。在此可以安全地聊天。",
```
原文：**This server has No Chat Reports installed, or prevents your messages from being reportable in some other known way. You can chat safely.**

---

该服务器使用了No Chat Reports模组或其他已知方式禁用了聊天举报，可以在此放心聊天。

---

```suggestion
	"gui.nochatreports.safety_status.secure": "该服务器使用了No Chat Reports模组或其他已知方式禁用了聊天举报，可以在此放心聊天。",
```